src/Enforce_Policies.php

```diff
@@ -66,36 +69,58 @@ public function enforce() {
 		$option   = $this->policies_setting->get();
 		$headers  = getallheaders();
 
-		if ( ! static::$enforced
+		if ( 0 === $this->enforced_policies++
```

Comment:
I rather put the increment as a separate statement (where the previous `$enforced = true` was).